src/test/java/org/apache/commons/text/similarity/LongestCommonSubsequenceTest.java

```diff
@@ -106,6 +108,8 @@ void testLongestCommonSubsequence() {
         assertEquals("", subject.longestCommonSubsequence("", ""));
         assertEquals("", subject.longestCommonSubsequence("left", ""));
         assertEquals("", subject.longestCommonSubsequence("", "right"));
+        assertEquals("", subject.longestCommonSubsequence("l", "a"));
+        assertEquals("", subject.longestCommonSubsequence("left", "a"));
         assertEquals("fog", subject.longestCommonSubsequence("frog", "fog"));
         assertEquals("", subject.longestCommonSubsequence("fly", "ant"));
```

Comment:
This doesn't improve the coverage beyond what we already have: 92%. What does this test that the current test doesn't? Unless you can explain it, please remove these changes. TY.